### DISCUSSION TITLE
Error initializing QuantumRenderer: no suitable pipeline found

### DISCUSSION THREAD

**Discussion by MikeVerba** [redfx-quantum/strange#128] [Q&A]
Caused by: java.lang.RuntimeException:  no suitable pipeline found

I'm trying to run JavaFX examples from Quantum Computing book repo:

https://github.com/johanvos/quantumjava

I've found similar issue here:
https://github.com/javafxports/openjdk-jfx/issues/237

I'm trying to run it on:
22.2.0 Darwin Kernel Version 22.2.0: Fri Nov 11 02:04:44 PST 2022; root:xnu-8792.61.2~4/RELEASE_ARM64_T8103 arm64

How can I fix this?

**Comment by DutchDave** [comment]
I was trying to run on a macbook. Got an error: with NSInternalInconsistencyException
Fixed it as suggested on https://stackoverflow.com/a/77967858/29283562

Added to the pom:
<dependency>
            <groupId>org.openjfx</groupId>
            <artifactId>javafx-controls</artifactId>
            <version>22.0.1</version>
        </dependency>
        <dependency>
            <groupId>org.openjfx</groupId>
            <artifactId>javafx-fxml</artifactId>
            <version>22.0.1</version>
        </dependency>
        
Maybe this can be fixed in strangefx pom?
